drivers/hmis_external_apis/app/models/hmis_external_apis/tc_hmis/importers/loaders/custom_assessment_loader.rb

```diff
@@ -104,7 +117,7 @@ def create_assessment_records(rows)
 
         assessment_date = row_assessment_date(row)
         # derived "last updated" timestamp for 9am on AssessmentDate
-        last_updated_timestamp = assessment_date.beginning_of_day.to_datetime + 9.hours
+        last_updated_timestamp = parse_date(assessment_date).beginning_of_day.to_datetime + 9.hours
```

Comment:
whoops, thanks. it looks like all assessment loaders return a date from `row_assessment_date` except for the UHA loader, which returns a string. It looks  like it still gets persisted just fine, just flagging that.